### PR TITLE
changed readmefile

### PR DESCRIPTION
**Pull request by roman-lazunin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,5 @@ Hello World!
 ```
 
 Hi there `people`!
+
+How areya.
```
Comment (roman-lazunin) [inline README.md]: are you?*

Comment (Dimitrios-Dimparis) [inline README.md]: It supposed to be "How are you" instead you wrote "areya" slang words.. 👍

File: README.md [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
+File List
 ---------
 ```
-Hello World! 
+Hello World!
 ```
 
 Hi there `people`!
+
+How are you.
```
Comment (roman-lazunin) [inline README.md]: happy now?

Comment (Dimitrios-Dimparis) [reply README.md]: Yup well done! 💯

**Pull request merged**